src/hotspot/share/gc/g1/g1DirtyCardQueue.hpp

```diff
@@ -31,7 +31,7 @@
 #include "gc/g1/g1ConcurrentRefineStats.hpp"
 #include "gc/shared/ptrQueue.hpp"
 #include "memory/allocation.hpp"
-#include "memory/padded.hpp"
```

Comment:
There are other uses of padded in this file besides those in the Queue that are being moved to lockFreeQueue.hpp.